src/element-style-interface.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Let's see if we can remove the need for this by simply having each of the libraries implement the interface on SnadyCSS and cooperate behind the scenes if necessary. This allows users to load whichever of these libraries and always write the same code.